Lmod-UGent.spec

```diff
@@ -1,8 +1,8 @@
 %global macrosdir %(d=%{_rpmconfigdir}/macros.d; [ -d $d ] || d=%{_sysconfdir}/rpm; echo $d)
 
 Name:           Lmod
-Version:        8.4.2
-Release:        1.ug%{?dist}
+Version:        8.4.10
+Release:        2.ug%{?dist}
```

Comment:
Why bump both the version and the release?

Reply:
because I built an RPM with a `.1` release number while testing, but then had to make another minor change (see 990f15e)

`Lmod-8.4.10-2.ug.el7.x86_64` is currently what's installed on `gligar06`